server/views/pages/explore.njk

```diff
@@ -23,9 +23,72 @@
     </div>
   </div>
 
+  {# Theme #}
   <div class="row">
     <div class="container">
       <div class="grid">
+        <div class="{{ {s:4, m:8, l:12, xl: 12} | gridClasses }}">
+          <h2 class="heading--large no-v-margin">A drop in the ocean</h2>
```

Comment:
![screen shot 2017-02-16 at 12 10 26](https://cloud.githubusercontent.com/assets/1394592/23020911/2c4299c0-f441-11e6-90e5-dc2c998a450b.png)

I feel like this thing that's being made up of root-scope classes would be better off as a component in its own right? Not sure, but it doesn't seem to fit well with what we've got so far.